android/titanium/src/java/org/appcelerator/titanium/TiApplication.java

```diff
@@ -1020,4 +1021,35 @@ public AccessibilityManager getAccessibilityManager()
 	public void verifyCustomModules(TiRootActivity rootActivity)
 	{
 	}
+
+	public void popToRootWindow()
+	{
+		/**
+		 * emulates the iOS Tab.popToRootWindow() by closing all windows above a TabGroup.
+		 */
+		int tabGroupPosition = -1;
+		boolean isTabGroup = false;
+
+		for (int i = 0; i <= activityStack.size(); ++i) {
```

Comment:
Can you confirm that the "close" events are behaving the same as on iOS?

Reply:
I would need to check iOS but it will fire all `close` events of each window if you call `popToRootWindow` in the example above. The log will show multiple `closing window` . But I'll run it on iOS to see if it is the same in a bit

Reply:
fixed the example so it will run on iOS too and yes, both show `closing window` (close even) for all opened windows.